content/stub.completion-ui.js

```diff
@@ -84,15 +84,16 @@ ContactIdentityCompleter.prototype = {
     }
     // and since we can now map from contacts down to identities, map contacts
     //  to the first identity for them that we find...
-    matches = [val.NOUN_ID == Gloda.NOUN_IDENTITY ? val : val.identities[0]
-               for each ([iVal, val] in Iterator(contactToThing))];
+    matches = Array.prototype.map.call(contactToThing, function(val) {
+      return val.NOUN_ID == Gloda.NOUN_IDENTITY ? val : val.identities[0];
+    });
 
     let rows = [asToken(
                   match.pictureURL(),
                   match.contact.name != match.value ? match.contact.name : null,
                   match.value,
                   match.value
-                ) for each ([iMatch, match] in Iterator(matches))];
+                ) for ([iMatch, match] of matches)];
```

Comment:
Amendment: since `matches` is an array, then you want `for match of matches`. The elements in your array are not pairs, I think...